lua/nvim-possession/config.lua

```diff
@@ -1,11 +1,20 @@
 local M = {}
 
 M.sessions = {
-	sessions_path = vim.fn.stdpath("data") .. "/sessions/",
+	-- sessions_path = vim.fn.stdpath("data") .. "/sessions/",
+	sessions_path = "/home/singularis/.config/nvim/misc/sessions/",
```

Comment:
This should probably default back to `vim.fn.stdpath("data") .. "/sessions/"` :p